### ISSUE TITLE
mdns advertising

### ISSUE THREAD

**Issue by ryanramage** [maxogden/ScreenCat#8]
Would be cool if you could advertise/see/connect to others on the same network via mdns like in mafintosh/airpaste

**Comment by maxogden** [comment]
YES I was thinking this actually. should be pretty straightforward to do!

**Comment by maxogden** [comment]
I think the way to implement this would be to check airpaste first for an e.g. cat-lobby channel. If it finds a peer it should use airpaste as the transport, and then fall back to cat-lobby

**Comment by hiccup01** [comment]
Bonjour anyone?
